### PR TITLE
Clean up of Microsoft.ApplicationModel.Resources.PriGen.targets (round 2)

### PR DESCRIPTION
**Pull request by rohanp-msft**
Remove some more unused Appx* properties and items in Microsoft.ApplicationModel.Resources.PriGen.targets.

Microsoft.ApplicationModel.Resources.PriGen.targets was created from Microsoft.AppxPackage.targets where a bunch of appx generation MSBuild script code was removed (Microsoft.AppxPackage.targets does more than just PRI generation, which is all that we want Microsoft.ApplicationModel.Resources.PriGen.targets to do, as the name suggests). At that point though, not all non-PRI gen related MSBuild script code was removed. I'll now try to remove it over several rounds. This is round 2.

Round 1: https://github.com/microsoft/ProjectReunion/pull/664

**How verified**
Used the MRTCore CPP sample app. Got rid of build outputs. Replaced the prigen targets file in the nuget package install with the one in this change. Built and ran the app.

### CODE REVIEW

**Review by axelandrejs — APPROVED** [review]

File: dev/MRTCore/packaging/Microsoft.ApplicationModel.Resources.PriGen.targets [hunk 1]
```diff
@@ -1435,44 +1245,6 @@
...
-
-  <PropertyGroup>
-    <_CreateAppxBundlePlatformSpecificArtifactsDependsOn>
-      $(Before_CreateAppxBundlePlatformSpecificArtifacts)
-
-      _GetPackageProperties;
-
-      _GetDefaultResourceLanguage;
-
-      _CreatePriConfigXmlForSplitting;
```
Comment (huichen123) [inline dev/MRTCore/packaging/Microsoft.ApplicationModel.Resources.PriGen.targets:1450]: >_CreatePriConfigXmlForSplitting; [](start = 6, length = 32)

are these tasks still called?

Comment (rohanp-msft) [reply dev/MRTCore/packaging/Microsoft.ApplicationModel.Resources.PriGen.targets:1450]: some may not be, i'll remove them in round 3

Comment (huichen123) [reply dev/MRTCore/packaging/Microsoft.ApplicationModel.Resources.PriGen.targets:1450]: for those still need to be run, are they still running when you remove this?

Comment (rohanp-msft) [reply dev/MRTCore/packaging/Microsoft.ApplicationModel.Resources.PriGen.targets:1450]: yup, e.g., _GetDefaultResourceLanguage

Comment (huichen123) [reply dev/MRTCore/packaging/Microsoft.ApplicationModel.Resources.PriGen.targets:1450]: any task in the list with PRI needs to run, e.g. _ExpandMainPriFile?

Comment (rohanp-msft) [reply dev/MRTCore/packaging/Microsoft.ApplicationModel.Resources.PriGen.targets:1450]: I suspect that any target in this list that's not referenced by another target in this file can go.

**Pull request merged**